### PR TITLE
[easy][utils][dense_polynomial] improve doc + code

### PR DESCRIPTION
**Pull request by mimoo**
as the title says

### CODE REVIEW

File: utils/src/dense_polynomial.rs [hunk 1]
```diff
@@ -22,7 +22,10 @@ pub trait ExtendedDensePolynomial<F: Field> {
     /// This function evaluates polynomial in chunks.
     fn eval(&self, elm: F, size: usize) -> Vec<F>;
 
-    /// Multiplies the chunks of a polynomial with powers of zeta^n
+    /// Multiplies the chunks of a polynomial with powers of zeta^n to make it of degree n.
```
Comment (querolita) [inline utils/src/dense_polynomial.rs:25]: I am not sure I understand the reasoning. 
If `f(x) = f0(x) + x^n * f1(x) + x^{2n} * f2(x)` and all three `f0, f1, f2` have degree n-1, then I understand that `f` has degree `n-1+2n = 3n-1`.

**Review by querolita — COMMENTED** [review]
I have one comment, once solved I will get back to the PR and accept

File: utils/src/dense_polynomial.rs (hunk 1 above)
Comment (mimoo) [reply utils/src/dense_polynomial.rs:25]: that's correct, that's why we do:

*`x^n` -> `z^n`
* `z^2n` -> `z^2n`

Comment (mimoo) [reply utils/src/dense_polynomial.rs:25]: probable the comment here should say "of degree n-1"

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**